README.md

```diff
@@ -16,44 +14,77 @@ Support for:
 * Negative frame numbers: -10-100
 * Padding: #=4 padded, @=single pad
 
-FrameSets
-=========
+## FrameSets
 
 A FrameSet wraps a sequence of frames in a list list container.
 
-Iterate a FrameSet
-------------------
-
-```
+### Iterate a FrameSet
+```python
 fs = fileseq.FrameSet("1-5")
 for f in fs:
   print f
 ```
 
-Random Access
--------------
+### Access Frames
 
+#### Using Indecies:
+```python
+>>> fs = fileseq.FrameSet("1-100:8")
+>>> fs[0] # First frame.
+1
+>>> fs[-1] # Last frame.
+98
 ```
-fs = fileseq.FrameSet("1-100:8")
-print fs[-1] # Print last frame
+
+#### Using Convenience Methods:
+```python
+>>> fs = fileseq.FrameSet("1-100:8")
+>>> fs.start() # First frame.
+1
+>>> fs.end() # Last frame.
+98
 ```
 
-FileSequence
-============
+## FileSequence
 
-```
+### Instantiate from String
+```python
 fileseq.FileSequence("/foo/bar.1-10#.exr")
 ```
 
-Finding Sequences on Disk
-=========================
+### Format Path for VFX Software
 
+```python
+>>> seq = fileseq.FileSequence("/foo/bar.1-10#.exr")
+>>> ''.join([seq.directory(), seq.basename(), seq.padding(), seq.extension()]) 
+"/foo/bar.#.exr"
```

Comment:
Maybe if you are showing an example of formatting the paths, you should also include an example of using the `FileSequence.format()` method?